_data/toc.yaml

```diff
@@ -1190,6 +1240,58 @@ manuals:
       title: Dev Environments (Preview)
     - path: /desktop/extensions/
       title: Extensions (Preview)
+    - sectiontitle: Extensions SDK (Preview)
```

Comment:
Just checking if this should be changed from 'Preview' to 'Beta'. I have another PR to be merged tomorrow that changes the user manual docs to 'Beta'.

Reply:
I can change both in this PR, to avoid conflicts...

Reply:
That'd be great, thank you.

Reply:
Also mentioned Beta in the reference section